_config.yml

```diff
@@ -14,4 +14,7 @@ plugins:
 
 # Optional. The default date format, used if none is specified in the tag.
 last-modified-at:
-    date-format: '%d/%m/%Y'
\ No newline at end of file
+    date-format: '%d/%m/%Y'
+
+# Color Theme
+color_scheme: light
```

Comment:
Possible to set based on user preferences/browser/OS?

Reply:
It should be possible, take a look at the .js file where it stores them, I wanted to do cookie based approach, but the issue is that ruby doesn't completely support it  so it would have to be written separately.

Reply:
The following there sets the global Initial Color_Scheme of the page, per say Default

Reply:
In that case just make a note somewhere, can merge as is for now